src/App.tsx

```diff
@@ -1,24 +1,299 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { addTodo, deleteTodo, getTodos } from './api/todos';
 
-const USER_ID = 0;
+const USER_ID = 11361;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [updatedTodos, setUptatedTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<string>('');
+  const [sortBy, setSortBy] = useState<string>('all');
+  const [isTodoDeleted, setIsTodoDeleted] = useState(false);
+  const [todosNotCompleted, setTodosNotCompleted] = useState(0);
+  const [isLoading, setIsLoading] = useState(false);
+  const [selectedTodo, setSelectedTodo] = useState<number[]>([]);
+  const [inputValue, setInputValue] = useState<string>('');
+  const [isLoadingTodo, setIsLoadingTodo] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+
+  useEffect(() => {
+    setIsLoading(true);
+    async function fetchTodos() {
+      try {
+        const data = await getTodos(USER_ID);
+
+        setTodos(data);
+        setUptatedTodos(data);
+        setTodosNotCompleted(
+          data.filter(todo => todo.completed === false).length,
+        );
+      } catch (error) {
+        setErrorMessage('Unable to load a todo');
+      } finally {
+        setIsLoading(false);
+      }
+    }
+
+    fetchTodos();
+    setIsTodoDeleted(false);
+  }, [isTodoDeleted, tempTodo]);
+
+  useEffect(() => {
+    if (errorMessage) {
+      setTimeout(() => {
+        setErrorMessage('');
+      }, 3000);
+    }
+  }, [errorMessage]);
+
+  useEffect(() => {
```

Comment:
This whole useEffect is redundant, implement similar logic like in your previous task `react_dynamic-list-of-todos`,
where you filtered todos in `TodoList`